### PR TITLE
feat: Add Kotlin support for android

### LINKED ISSUE #80: Add kotlin support in android

### PR DESCRIPTION
**Pull request by talwinder50**
https://user-images.githubusercontent.com/7862595/203861701-7804ade6-b551-47c5-8245-7af5c26df7fd.mp4

closes #80

Signed-off-by: Talwinder kaur <talwinder.kaur@avast.com>

### CODE REVIEW

File: demo/app/android/app/src/main/java/dev/trustbloc/wallet/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,105 @@
+package dev.trustbloc.wallet
```
Comment (rolsonquadras) [inline demo/app/android/app/src/main/java/dev/trustbloc/wallet/MainActivity.kt:1]: The file path includes `java`. Should this be changed ?

**Comment by rolsonquadras** [comment]
We need to consider following generated plugin file:
https://github.com/trustbloc/wallet-sdk/blob/34e4f2b94dea2cd76c06c889d42d75139835836a/demo/app/android/app/src/main/AndroidManifest.xml#L29

File: demo/app/android/app/src/main/java/dev/trustbloc/wallet/MainActivity.kt (hunk 1 above)
Comment (talwinder50) [reply demo/app/android/app/src/main/java/dev/trustbloc/wallet/MainActivity.kt]: Its updated

**Review by rolsonquadras — COMMENTED** [review]
Looks like images are reverted to flutter default image - see ic_launcher.png file

**Comment by talwinder50** [comment]
> Looks like images are reverted to flutter default image - see ic_launcher.png file.
> 
> Can you try to run `flutter packages pub run flutter_launcher_icons:main` ?

updated

**Review by vkubiv — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**